### DISCUSSION TITLE
Investigate using pydantic/datafusion-query-cache

### DISCUSSION THREAD

**Discussion by gaby** [openobserve/openobserve#4715] [Ideas]
This crate is on it's early stages, but has potential to benefit `OpenObserve`.

https://github.com/pydantic/datafusion-query-cache

**Comment by prabhatsharma** [comment]
Query result cache has already been implemented. It has been available for a while.

**Comment by gaby** [comment]
Thanks

**Comment by hengfeiyang** [comment]
We have the env:
```
ZO_RESULT_CACHE_ENABLED=false
```

You can enable it by set it to `true`.

**Reply by gaby** [reply]
Thanks, I will enable it for the next release. 💪
